src/lists.py

```diff
@@ -52,7 +68,12 @@ def drop(x: LList[T], k: int) -> LList[T]:
     >>> drop(Link(1, Link(2, None)), 1)
     Link(2, None)
     """
-    ...
+    if x is None or x.tail is None:
+        return None
+    while k > 0 and x.tail is not None:
+        x = x.tail
+        k -= 1
+    return x
```

Comment:
What's wrong with this?

```python
    while k > 0 and x is not None:
        x = x.tail
        k -= 1
    return x
```

Why look at `x.tail` instead of `x`?